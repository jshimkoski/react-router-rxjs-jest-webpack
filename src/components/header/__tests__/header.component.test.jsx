import React from 'react';
import Header from '../header.component';
import renderer from 'react-test-renderer';

describe('Header', () => {
  let component;
  beforeEach(() => {
    component = renderer.create(
      <Header/>
    ).toJSON();
  });

  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });
});