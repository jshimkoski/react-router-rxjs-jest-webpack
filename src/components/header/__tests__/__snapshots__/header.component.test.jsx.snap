exports[`Header renders correctly 1`] = `
<header>
  <h1>
    Header
  </h1>
  <nav
    className="nav">
    <img
      src="test-file-stub" />
    <ul>
      <li>
        <a
          onClick={[Function]}
          style={Object {}}>
          Home
        </a>
      </li>
      <li>
        <a
          onClick={[Function]}
          style={Object {}}>
          About
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
